packages/eslint-plugin-lit-a11y/tests/lib/rules/role-has-required-aria-attrs.js

```diff
@@ -30,8 +30,9 @@ ruleTester.run('role-has-required-aria-attrs', rule, {
         'html`<span role="checkbox" aria-checked="false" aria-labelledby="foo" tabindex="0"></span>`',
     },
     { code: 'html`<span role="row"></span>`' },
+    { code: 'html`<span role="${foo}"></span>`' },
+    { code: 'html`<span role="checkbox" aria-checked=${true}></span>`' },
     { code: 'html`<input type="checkbox" role="switch" aria-checked="true" />`' },
-    { code: 'html`<div role="combobox" aria-controls="foo"  aria-expanded="foo"></div>`' },
```

Comment:
Why remove what looks like a valid combo box usage? Seems like we’d want to test this...

Reply:
Can we add this as `<div role="combobox"  aria-expanded="foo"></div>` instead of removing?